appveyor/scripts/deployScript.ps1

```diff
@@ -31,6 +31,34 @@ if (!$env:APPVEYOR_PULL_REQUEST_NUMBER -and $env:versionType) {
 		# The new address must be also included in appveyor\ssh_known_hosts in this repo
 		# Otherwise ssh will freeze on input!
 		cat deploy.json | ssh nvaccess@deploy.nvaccess.org nvdaAppveyorHook
```

Comment:
is this line still relevant?

Reply:
This should be removed once the new system becomes the source of truth.